test/integration/fake_upstream.h

```diff
@@ -150,12 +150,14 @@ class QueuedConnectionWrapper : public Network::ConnectionCallbacks {
   void set_parented() {
     std::unique_lock<std::mutex> lock(lock_);
     parented_ = true;
+    ENVOY_LOG_MISC(debug, "HTD parented! {}", static_cast<void*>(this));
```

Comment:
Did you mean to leave this line and below in? Doesn't seem that useful.

Reply:
Will fix.